nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/tasks.js

```diff
@@ -2,4 +2,7 @@
 
 module.exports = function () {
   this.Then('I can see the list of tasks', () => this.ui.drawer.tasks.isVisible().should.be.true);
+  this.Then('I can see the View Tasks Dashboard link', () => this.ui.tasksLink.isVisible().should.be.true);
+  this.When('I click the View Tasks Dashboard link', () => this.ui.tasksLink.click());
+  this.Then('I can see the Tasks Dashboard', () => this.ui.tasksDashboard.isVisible().should.be.true);
```

Comment:
Tried both solutions but it turns out that the `#tasks-dashboard` is not located in `<nuxeo-home>`.